src/test/resources/fixtures/getBeaconResponse.json

```diff
@@ -0,0 +1,80 @@
+{
+  "meta": {
+    "count": 1,
+    "pageSize": 1
+  },
+  "data": [
+    {
+      "type": "beacon",
+      "id": "97b306aa-cbd0-4f09-aa24-2d876b983efb",
+      "attributes": {
+        "hexId": "Hex me",
+        "status": "NEW",
+        "manufacturer": "Ocean Signal",
+        "createdDate": "2020-02-01T00:00",
+        "model": "EPIRB1",
+        "manufacturerSerialNumber": "1407312904",
+        "chkCode": "9480B",
+        "batteryExpiryDate": "2020-02-01T00:00",
+        "lastServicedDate": "2020-02-01T00:00"
+      },
+      "relationships": {
+        "uses": {
+          "data": [{ "type": "beaconUse", "id": "1" }]
+        },
+        "owner": {
+          "data": { "type": "beaconPerson", "id": "1" }
```

Comment:
even if we **know** that there is only ever one owner, should this be an array? Does the JSON API spec demand it? 🤔

Reply:
i have checked, it does not.
```

"relationships": {
      "author": {
        "links": {
          "self": "http://example.com/articles/1/relationships/author",
          "related": "http://example.com/articles/1/author"
        },
        "data": { "type": "people", "id": "9" }
      },
      "comments": {
        "links": {
          "self": "http://example.com/articles/1/relationships/comments",
          "related": "http://example.com/articles/1/comments"
        },
        "data": [
          { "type": "comments", "id": "5" },
          { "type": "comments", "id": "12" }
        ]
      }
    }
```

Reply:
Yeah I wasn't too sure what would be best. 

I don't think JSON API spec demands it - in their example [on their site](https://jsonapi.org/) they have `author` with just a single `data`:

```
    "relationships": {
      "author": {
        "links": {
          "self": "http://example.com/articles/1/relationships/author",
          "related": "http://example.com/articles/1/author"
        },
        "data": { "type": "people", "id": "9" }
      },
```

I don't know 💯 if beacons only have one owner either... 🤔

Reply:
Snap ✋